opencti-platform/opencti-front/src/private/components/analyses/malware_analyses/MalwareAnalysisEditionDetails.tsx

```diff
@@ -178,6 +205,15 @@ const MalwareAnalysisEditionDetails: FunctionComponent<MalwareAnalysisEditionDet
         values,
       }) => (
         <Form style={{ margin: '20px 0 20px 0' }}>
+          {(!checkConfidenceForEntity(malwareAnalysis) && (
+            <Alert severity="warning" variant="outlined"
+              style={{ marginTop: 20, marginBottom: 20 }}
+            >
+              {t_i18n(
+                'Your maximum confidence level is insufficient to edit this object.',
+              )}
+            </Alert>
+          ))}
```

Comment:
I think you should wrap this into a dedicated component <AlertConfidenceForEntity />

That uses internally the hook useConfidenceLevel and simply returns null if check is ok.